### PR TITLE
Add brazilian portuguese (pt-BR) translation

### PR DESCRIPTION
**Pull request by DaniloPedrazzi**
Made to all my friends using the app who couldn't read english :D

### CODE REVIEW

**Comment by Sriramjerry120** [comment]
Automatically music app stop how to this any ideas

**Comment by DaniloPedrazzi** [comment]
> Automatically music app stop how to this any ideas

I couldn't understand well... Are you requesting a feature?

File: app/src/main/res/values-pt-rBR/strings.xml
Comment (dtomvan) [inline app/src/main/res/values-pt-rBR/strings.xml:1]: Filename values-pt-**r**BR/strings.xml. Is that correct?

**Review by dtomvan — COMMENTED** [review]
I don't speak portugese, but LGTM, except for that filename.

If it builds we can merge.

File: app/src/main/res/values-pt-rBR/strings.xml
Comment (DaniloPedrazzi) [reply app/src/main/res/values-pt-rBR/strings.xml:1]: Yeah, this "r" before the BR is for "region", a naming convention for specific languages like Brazilian portuguese or Canadian French.

**Comment by DaniloPedrazzi** [comment]
Oh no I closed it by accident...

New PR: #620

**Pull request closed without merging**